src/patternfly/demos/DataList/data-list-content-2.hbs

```diff
@@ -9,11 +9,11 @@
   {{/l-flex}}
   {{#> l-flex l-flex--modifier="pf-m-wrap"}}
     {{> data-list--icon-text data-list--icon-text--icon="code-branch" data-list--icon-text--text="5"}}
-    {{> data-list--icon-text data-list--icon-text--icon="code" data-list--icon-text--text="9"}}
-    {{> data-list--icon-text data-list--icon-text--icon="cube" data-list--icon-text--text="2"}}
-    {{> data-list--icon-text data-list--icon-text--icon="check-circle" data-list--icon-text--text="11"}}
-    {{> data-list--icon-text data-list--icon-text--icon="exclamation-triangle" data-list--icon-text--text="4"}}
-    {{> data-list--icon-text data-list--icon-text--icon="times-circle" data-list--icon-text--text="1"}}
+    {{> data-list--icon-text data-list--icon-text--icon="code" data-list--icon-text--text="9" data-list--icon-text--screenreader-text="Code blocks"}}
+    {{> data-list--icon-text data-list--icon-text--icon="cube" data-list--icon-text--text="2" data-list--icon-text--screenreader-text="Workspaces"}}
+    {{> data-list--icon-text data-list--icon-text--icon="check-circle" data-list--icon-text--modifier="pf-m-success" data-list--icon-text--text="11" data-list--icon-text--screenreader-text="Completed"}}
```

Comment:
What you have here is great and probably preferred, but just FYI in cases like this where the `data-list--icon-text` is so limited in scope (it should only have a single icon in it), it's totally fine to just use `icon-content--modifier` directly on `data-list--icon-text`.

```suggestion
    {{> data-list--icon-text data-list--icon-text--icon="check-circle" icon-content--modifier="pf-m-success" data-list--icon-text--text="11" data-list--icon-text--screenreader-text="Completed"}}
```

Also seeing this makes me think we should really enhance the icon component hbs to have parameters/props for `icon--IsWarning`, `icon--IsSmall`, etc